rfc8826.xml

```diff
@@ -834,32 +937,53 @@ the title) for use on https://www.rfc-editor.org/search. -->
           <section anchor="sec.sas" numbered="true" toc="default">
             <name>Short Authentication Strings</name>
             <t>
-              ZRTP <xref target="RFC6189" format="default"/> uses a "short authentication string" (SAS) which is derived
-              from the key agreement protocol. This SAS is designed to be compared
+              ZRTP <xref target="RFC6189" format="default"/> uses a "Short Authentication String" (SAS) which is derived
+              from the key agreement protocol.
+
+<!-- [rfced] Section 4.3.2.2:  Is the SAS always derived from the key
+agreement protocol (in which case "(SAS), which is" would be correct)
+or only sometimes derived from the key agreement protocol (in which
+case "(SAS) that is" would be correct)?
```

Comment:
This is correct as is. One could imagine a short authentication string which was not derived from the AKE, one supposes.